bin/semantic-release.js

```diff
@@ -25,7 +25,7 @@ See https://github.com/semantic-release/semantic-release/blob/master/docs/suppor
 
 execa("git", ["--version"])
   .then(({ stdout }) => {
-    const gitVersion = findVersions(stdout)[0];
+    const gitVersion = findVersions(stdout, { loose: true })[0];
```

Comment:
so we capture details that we can look back to in the future, could you mention the details that lead to this change?

Reply:
The output of `git -v` on Windows at my current install is: `git version 2.39.0.windows.2`.  This is not fully semver compliant.  Issue #https://github.com/git-for-windows/git/issues/4086 was opened (and closed) recently regarding this (tl;dr it is not going to change). It appears the semver package works and the find-versions package does not.  The `loose` option extracts the version info from the string correctly.  The non-Windows versions could have similar issues in the future that this will hopefully avoid.  `find-versions` could be updated but a rejection would not be unexpected.